ui/src/api/base58id.js

```diff
@@ -0,0 +1,15 @@
+const base58characters = '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz'.split('');
+
+/**
+ * Generate a readable unique id
+ * @param length
+ * @returns {string}
+ */
+module.exports = (length) => {
+  let s = '';
+  for (let i = 0; i < length; i++) {
+    const c = Math.floor(Math.random() * 58);
```

Comment:
could also use crypto.randomUUID() ? it is readable standard but just doesn't use the base58...